src/routes/safe/components/Apps/components/SignMessageModal/SignMessageModal.test.tsx

```diff
@@ -114,10 +118,62 @@ describe('SignMessageModal Component', () => {
         onUserConfirm={jest.fn()}
         onTxReject={jest.fn()}
         requestId="1"
+        method={Methods.signMessage}
         app={getEmptySafeApp()}
       />,
     )
 
     expect(screen.getByText(transactionHash)).toBeVisible()
   })
+
+  test('If message is typed data, displays the message correctly', () => {
```

Comment:
the test doesn't check if the message is displayed

Reply:
Updated ;)